en/deploy/README.md

```diff
@@ -1,246 +1,304 @@
+
+So we initialise the database on the server just like we did the one on your own computer, with `migrate` and `createsuperuser`:
+
+
+    (mvenv)20:20 ~ $ python manage.py migrate
+    Operations to perform:
+    [...]
+      Applying sessions.0001_initial... OK
+
+
+    (mvenv)20:20 ~ $ python manage.py createsuperuser
+
+
+## Publishing our blog as a web app
+
+Now our code is on PythonAnywhere, our virtualenv is ready, the static files are collected, and the database is initialised, we're ready to publish it as a web app.
+
+Click back to the PythonAnywhere dashboard by clicking on its logo, and go click on the **Web** tab, and hit **Add a new web app**.
+
+In the dialog, after confirming your domain name, choose **manual configuration** (NB *not* the "Django" option). Next, choose **Python 3.4**, and click Next to finish the wizard.  
+
+> **Note** make sure you choose the "Manual configuration" option, not the "Django" one. We're too cool for the default PythonAnywhere Django setup ;-)
+
+
+### Setting the virtualenv
+
+You'll be taken to the PythonAnywhere config screen for your webapp, which is where you'll need to go whenever you want to make changes to the app on the server.
 
-We'll be making your blog available on the Web at `[your blog's name].herokuapp.com`, so we need to choose a name that nobody else has taken. This name doesn't need to be related to the Django `blog` app or to `mysite` or anything we've created so far. The name can be anything you want, but Heroku is quite strict as to what characters you can use: you're only allowed to use simple lowercase letters (no capital letters or accents), numbers, and dashes (`-`).
 
-Once you've thought of a name (maybe something with your name or nickname in it), run this command, replacing `djangogirlsblog` with your own application name:
+<img src="images/pythonanywhere_web_tab_virtualenv.png" />
 
-    $ heroku create djangogirlsblog
+In the "Virtualenv" section, click the red text that says "Enter the path to a virtualenv", and enter:  */home/yourusername/my-first-blog/myvenv/*
 
-> __Note__: Remember to replace `djangogirlsblog` with the name of your application on Heroku.
+> __Note__: substitute your own username as appropriate. If you make a mistake, PythonAnywhere will show you a little warning.
 
-If you can't think of a name, you can instead run
 
-    $ heroku create
+### Configuring the WSGI file
 
-and Heroku will pick an unused name for you (probably something like `enigmatic-cove-2527`).
+Django works using the "WSGI protocol", a standard for serving websites using Python, which PythonAnywhere supports.  The way we configure PythonAnywhere to recognise our django blog is by editing a WSGI configuration file.  
 
-If you ever feel like changing the name of your Heroku application, you can do so at any time with this command (replace `the-new-name` with the new name you want to use):
+Click the link to it at the top of the web tab (it'll be named something like 
```

Comment:
"at the top" is a little to vague IMO, I thought it was the link next to "Configuration for". I would say that way: "Click on the "WSGI configuration file" link (in the "Code" section towards the top of the page)".